webapp/src/css/targets.less

```diff
@@ -8,22 +8,23 @@
   .target {
     width: 296px;
     min-height: 151px;
-    background-color: @gray-ultra-light;
+    background-color: @target-card-background-color;
     border: 1px solid @gray-light;
-    border-radius: @radius-size;
-    margin: 15px;
+    border-radius: @radius-size-medium;
+    margin: 10px 12px;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
 
     .heading {
       display: flex;
-      padding: 10px;
+      padding: 20px;
       background-color: @white;
-      border-bottom-left-radius: @radius-size;
-      border-bottom-right-radius: @radius-size;
+      border-bottom-left-radius: @radius-size-medium;
+      border-bottom-right-radius: @radius-size-medium;
+      border-top: 1px solid @gray-light;
```

Comment:
Please create a variable in variable.less, we can't use private-vars directly [see warning](https://github.com/medic/cht-core/blob/b4a54c020ee9ed0239fb8ca004f1a428c1ea66d1/webapp/src/css/private-vars.less#L3)